common/autoware_universe_utils/include/autoware/universe_utils/math/accumulator.hpp

```diff
@@ -65,11 +65,11 @@ class Stat
   unsigned int count() const { return count_; }
 
   template <typename U>
-  friend std::ostream & operator<<(std::ostream & os, const Stat<U> & stat);
+  friend std::ostream & operator<<(std::ostream & os, const Accumulator<U> & accumulator);
 
 private:
   T min_ = std::numeric_limits<T>::max();
-  T max_ = std::numeric_limits<T>::min();
+  T max_ = std::numeric_limits<T>::lowest();
```

Comment:
I found a bug here we should use lowest instead of min, I also found there are many similar mistakes in the whole autoware.universe, I will remind others to fix them.